smartmin/management/tests.py

```diff
@@ -37,18 +37,45 @@ def test_command(self, mock_write_dump, mock_load_migrations):
                 RunSQL("CREATE INDEX a_test ON foo(bar);"),
                 RunPython(mock_run_python)
             ]),
+            MockMigration(operations=[
+                RunSQL("CREATE OR REPLACE FUNCTION a_func(i INTEGER, s TEXT) RETURNS integer AS $$ BEGIN RETURN i + 1; END; $$ LANGUAGE plpgsql;"),
+
+                # different function because it has different parameters
+                RunSQL("CREATE OR REPLACE FUNCTION a_func(s text, integer) RETURNS text AS $$ BEGIN RETURN UPPER(s); END; $$ LANGUAGE plpgsql;"),
```

Comment:
integer here has not parameter name?

Reply:
that's valid sql.. included here in case collect_sql has to handle a case like that. AFAIK if can reference parameters as `$1` if using SQL instead of plpgsql